proposals/0448-type-variable-scoping.rst

```diff
@@ -551,10 +564,13 @@ Effects
    We expect a future proposal to remedy this problem, with either a modifier or some symbol.
    For example, perhaps we would say e.g. ``f (Just @(*a) x) = ...`` to denote an occurrence of already-in-scope type variable ``a``.
 
-#. Because ``-XScopedTypeVariables`` implies ``-XTypeAbstractions``,
-   people using ``-XScopedTypeVariables`` would have access to the new features without enabling a new extension.
-   This is backward-compatible with the current implementation, which requires both ``-XScopedTypeVariables`` and ``-XTypeApplications`` to be in effect.
-   (With this proposal, ``-XScopedTypeVariables`` alone would be enough.)
+#. Backward-compatibility with the current implementation,
+   which requires both ``-XScopedTypeVariables`` and ``-XTypeApplications`` to be in effect but nothing more,
+   is preserved.
+   But whenver this is used a deprecation warning will be issued.
+
+#. We have the option of removing the above implication (when is not proposed).
+   That will clean up new experimental functionality from leaking under established extensions.
```

Comment:
But it *is* proposed!  It's not an option, it's part of the proposed change spec.

Reply:
Oops! A holdover from when it wasn't.

Reply:
Unfortunately I couldn't leave a suggestion in this thread, but see what I did in 47c622cefa710e08e62e375af79c4eec638acfda.